pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/TransformOperator.java

```diff
@@ -86,13 +86,14 @@ protected TransferableBlock getNextBlock() {
     try {
       TransferableBlock block = _upstreamOperator.nextBlock();
       return transform(block);
-    } catch (Exception e) {
+    } catch (RuntimeException e) {
```

Comment:
Should we catch it or just throw it out?

Reply:
I don't actually know what we want or should do here. I'm keeping the old behavior but being more specific.

Previous code was catching `Exception` while no checked exception was thrown here. So I assumed that even older code was throwing a checked exception. Now that only runtime exceptions are thrown I think it is better to explicitly only catch `RuntimeException`s.